qa/L0_lifecycle/test.sh

```diff
@@ -373,15 +373,53 @@ wait $SERVER_PID
 LOG_IDX=$((LOG_IDX+1)) 
 
 
+# disable model polling, no model in the repository should be loaded
```

Comment:
Model polling is separate from lazy model and we can't change that behavior now. Even if polling is disabled we still need to load the model repository on startup. Perhaps --allow-model-control which defaults to false. We need to check in the parser to make sure that both are not enabled.